workspaces/web/src/components/ChallengeScreen.svelte

```diff
@@ -161,4 +161,7 @@
   .section {
     padding: 1.5em;
   }
+  .challenge {
+    padding: 2em 0;
```

Comment:
Why is challenge getting this padding?

Reply:
I tried to follow the design present here https://github.com/kantord/LibreLingo/issues/81#issuecomment-601997442
Little bit more spacing was introduced in the design for cleaner look so, I added padding in `challenge`.